bin/jsonwebtokenydoo.js

```diff
@@ -4,10 +4,13 @@ const inquirer = require("inquirer");
 const jwt = require("jsonwebtoken");
 const chalk = require("chalk");
 const axios = require("axios");
+const clipboardy = require("clipboardy");
 require("dotenv").config();
 
 const { findPrivateKey } = require("../lib/private-key");
 
+const BASE_URL = process.env.BASE_URL || "https://api.github.com"
```

Comment:
`jsonwebtokenydoo` is designed to work especially well when run in the working directory of a probot app. So I think we should support both `BASE_URL` but also `GHE_HOST` (since that is what Probot uses) and since `GHE_HOST` works a tiny differently